src/internal/methods/NeoArmMethod.h

```diff
@@ -204,10 +204,13 @@ template<typename T_SPEEDPROPS> class NeoArmMk20dxSpeedBase
         uint8_t pix;
         uint8_t mask;
 
-        volatile uint8_t* set = portSetRegister(pin);
-        volatile uint8_t* clr = portClearRegister(pin);
+        volatile auto set = portSetRegister(pin);
+        volatile auto clr = portClearRegister(pin);
 
         uint32_t cyc;
+        #if !defined(KINETIS)
+        uint32_t msk = digitalPinToBitMask(pin);
```

Comment:
why did the original use "1" hardcoded?  Is there not a `digitalPinToBitMask()` on the others and should this be changed for all?

Reply:
not hardcoded anymore as in your other suggestions.